src/Form/FormHelperText.js

```diff
@@ -0,0 +1,90 @@
+// @flow weak
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { createStyleSheet } from 'jss-theme-reactor';
+import withStyles from '../styles/withStyles';
+
+export const styleSheet = createStyleSheet('MuiFormHelperText', theme => ({
+  root: {
+    color: theme.palette.input.helperText,
+    fontFamily: theme.typography.fontFamily,
+    fontSize: 12,
+    lineHeight: 1.4,
+    margin: 0,
+  },
+  error: {
+    color: theme.palette.error.A400,
+  },
+  disabled: {
+    color: theme.palette.input.disabled,
+  },
+}));
+
+function FormHelperText(props, context) {
+  const {
+    children,
+    classes,
+    className: classNameProp,
+    disabled: disabledProp,
+    error: errorProp,
+    ...other
+  } = props;
+  const { muiFormControl } = context;
+
+  let disabled = disabledProp;
+  let error = errorProp;
+
+  if (muiFormControl && typeof disabled === 'undefined') {
+    disabled = muiFormControl.disabled;
+  }
+
+  if (muiFormControl && typeof error === 'undefined') {
+    error = muiFormControl.error;
+  }
+
+  const className = classNames(
+    classes.root,
+    {
+      [classes.disabled]: disabled,
+      [classes.error]: error,
+    },
+    classNameProp,
+  );
+
+  return (
+    <p className={className} {...other}>
+      {children}
+    </p>
+  );
+}
+
+FormHelperText.propTypes = {
+  /**
+   * The content of the component.
+   */
+  children: PropTypes.node,
+  /**
+   * Useful to extend the style applied to components.
+   */
+  classes: PropTypes.object.isRequired,
+  /**
+   * The CSS class name of the root element.
```

Comment:
We went with ignoring those from the documentation as quite idiomatic in the React world:
```js
  /**
   * @ignore
   */
```